pxtblocks/fields/field_variabledropdown.ts

```diff
@@ -0,0 +1,13 @@
+namespace pxtblockly {
+    export class FieldVariableDropdown extends Blockly.FieldVariable {
```

Comment:
LGTM, maybe give this a name that's a little more descriptive of the fact that it's a "variable dropdown minus delete option" eg "FieldVariableSelectDropdown", "FunctionArgumentVariableDropdown" or something.... like that but better? or you could just leave a comment explaining what it is

Reply:
changed the name to be FieldArgumentVariable, and added a comment!